index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="header">
+        <span class="logo"> <img src="" alt="My Logo"></span> 
+        <span class="hamburger"><img src="./img/hamburger.png" alt=""></span>
+    </div>
+    <main class="main">
```

Comment:
- [ ] Kindly for semantic reasons here make use of a `section` tag to wrap your headline instead of the `main` tap. Actually, it's correct to wrap everything with the `main` tag but each section inside the main should be in a `section` tag.

Reply:
@ridwanediallo 
You say it's ok to wrap everything inside the `main`. why then do I need `section` since my page clearly has just one section?